pulsar-client-api/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -187,6 +187,21 @@
      */
     ConsumerBuilder<T> ackTimeout(long ackTimeout, TimeUnit timeUnit);
 
+    /**
+     * Set the delay to wait before re-delivering messages that have failed to be process.
+     * <p>
+     * When application uses {@link Consumer#negativeAcknowledge(Message)}, the failed message
+     * will be redelivered after a fixed timeout. The default is 1 min.
```

Comment:
can we make this comment more accurate since that is not necessarily going to be true.  Messages can be redelivered anytime from 0 - timeout

Reply:
Agree

Reply:
Fixed the actual behavior